codecov.yml

```diff
@@ -0,0 +1,4 @@
+ignore:
```

Comment:
This is meant to correct the codecov.io report. Not working as planned and it remains to be seen why. At any rate, the code coverage report red blobs include us not having coverage for code only used by tests.